### PR TITLE
List Divider Simple (Wearable): update style according to guideline

### PR DESCRIPTION
**Pull request by grzegorz-czajkowski**
Two commits:
 * List Divider Simple (Wearable): update style according to guideline
 * List Divider Simple (example): use ui-listview

### CODE REVIEW

File: src/css/profile/wearable/changeable/theme-circle/listview.less [hunk 1]
```diff
@@ -236,55 +236,29 @@
...
-								opacity: 1;
-								height: 50 * @px_base;
-								right: 111 * @px_base;
-							}
+						// List divider "|".
+						&::after {
+							opacity: 1;
+							height: 50 * @px_base;
+							right: 111 * @px_base;
+							background: rgba(255, 255, 255, 0.19);
```
Comment (lslachciak) [inline src/css/profile/wearable/changeable/theme-circle/listview.less]: why not use 
@color_grid_list_divider_bg ?

File: examples/wearable/UIComponents/contents/presentation-views/list/list-divider/divider.html [hunk 2]
```diff
@@ -19,7 +19,7 @@
 <body>
 	<div class="ui-page" id="page">
 		<div class="ui-content">
-			<ul class="ui-arc-listview">
+			<ul class="ui-listview" >
```
Comment (DohyungLim) [inline examples/wearable/UIComponents/contents/presentation-views/list/list-divider/divider.html]: last 1 space remove.

File: src/css/profile/wearable/changeable/theme-circle/listview.less [hunk 3]
```diff
@@ -236,55 +236,29 @@
 
 				&.ui-li-grid-full {
 					justify-content: space-between;
+					min-height: 108 * @unit_base;
```
Comment (DohyungLim) [inline src/css/profile/wearable/changeable/theme-circle/listview.less]: use @px_base.

**Review by DohyungLim — COMMENTED** [review]
2 comment.

File: src/css/profile/wearable/changeable/theme-circle/listview.less (hunk 1 above)
Comment (grzegorz-czajkowski) [reply src/css/profile/wearable/changeable/theme-circle/listview.less]: Right. Fixed.

File: examples/wearable/UIComponents/contents/presentation-views/list/list-divider/divider.html (hunk 2 above)
Comment (grzegorz-czajkowski) [reply examples/wearable/UIComponents/contents/presentation-views/list/list-divider/divider.html]: Ok, removed.

File: src/css/profile/wearable/changeable/theme-circle/listview.less (hunk 3 above)
Comment (grzegorz-czajkowski) [reply src/css/profile/wearable/changeable/theme-circle/listview.less]: Fixed.

**Review by lslachciak — APPROVED** [review]
seems good, works well

**Pull request merged**